docs/source/posts/2023/2023-08-24-final-report-tvcastillod.rst

```diff
@@ -0,0 +1,175 @@
+- *Raymarching algorithm and lighting*: For the raymarching algorithm a small value of 20 was taken as the maximum distance since we apply the technique to each individual object and not all at the same time, additionally, we set the convergence precision to 0.001. We use the central differences method to compute the normals necessary for the scene’s illumination, besides the Blinn-Phong lighting technique which is high-quality and computationally cheap.
+- *Visualization example*: Below is a detailed visualization of the ellipsoids created from this new implementation.
+
+.. image:: https://user-images.githubusercontent.com/31288525/244503195-a626718f-4a13-4275-a2b7-6773823e553c.png
+    :width: 376
+    :align: center
+
+This implementation does show a better quality in the displayed glyphs, and support the display of a large amount of data, as seen in the image below. For this reason a tutorial was made to justify in more detail the value of this new implementation. Below are some images generated for the tutorial.
+
+.. image:: https://user-images.githubusercontent.com/31288525/260906510-d422e7b4-3ba3-4de6-bfd0-09c04bec8876.png
+    :width: 600
+    :align: center
+
+*Pull Requests:*
+
+-  **Ellipsoid actor implemented with SDF (Merged)** https://github.com/fury-gl/fury/pull/791
+-  **Tutorial on using ellipsoid actor to visualize tensor ellipsoids for DTI (Merged)** https://github.com/fury-gl/fury/pull/818
+
+**Future work:** In line with one of the initial objectives, it is expected to implement billboards later on, to improve the performance, i.e., higher frame rate and less memory usage for the tensor ellipsoid creation. In addition to looking for ways to optimize the naive raymarching algorithm and the definition of SDFs.
+
+Objectives in Progress
+----------------------
+
+DTI uncertainty visualization
+*****************************
+
+The DTI visualization pipeline is fairly complex, a level of uncertainty arises, which, if visualized, helps to assess the accuracy of the model. This measure is not currently implemented, and even though the are several methods to calculate a visualize the uncertainty in the DTI model, because of its simplicity and visual representation, we considered Matrix Perturbation Analysis (MPA) proposed by Basser [1]_. This measurement is visualized as double cones representing the variance of the main direction of diffusion, for which raymarching tecnique was also used in the creation of these objects.
```

Comment:
* `fairly complex, a level` -> `faily complex as a level`
* `and even though the are` -> `and even though there are`
* `can be approximated` -> `it can be approximated`